### ISSUE TITLE
Window component does not assign id to the title bar <span> element (inconsistent with Dialog)

### ISSUE THREAD

**Issue by yanmariomenev** [telerik/kendo-angular#4905]
## Bug Description

There is a potential accessibility issue in the Window component related to how the title bar markup is generated.
Currently, there is an inconsistency between the Dialog and Window components:
Issue

- Dialog automatically adds a  element with an id used for ARIA associations in the title bar.
Example:
[https://www.telerik.com/kendo-angular-ui/components/dialogs/dialog/title](https://www.telerik.com/kendo-angular-ui/components/dialogs/dialog/title)
- Window does not add an id to the  element in its title bar, which causes an accessibility mismatch.
Example:
[https://www.telerik.com/kendo-angular-ui/components/dialogs/window/title](https://www.telerik.com/kendo-angular-ui/components/dialogs/window/title)

This results in missing or incorrect ARIA attributes for the Window component, impacting accessibility tooling.

When the title bar in Window uses a [custom title bar](https://www.telerik.com/kendo-angular-ui/components/dialogs/window/service#specifying-custom-title-bar) and contains a , then the id is added:

```
<span class="k-window-title" id="kendo-window-title-708921">Custom TitleBar</span>
```

## Steps to Reproduce

Open one of our [examples](https://www.telerik.com/kendo-angular-ui/components/dialogs/window/title) and inspect the DOM.
Then check the [Dialog](https://www.telerik.com/kendo-angular-ui/components/dialogs/dialog/title), which adds the correct attribute on the span element.

## Expected Behavior

Id should be added similar how the Dialog component.

## Workaround

[https://stackblitz.com/edit/angular-4im6ksuc?file=src%2Fapp%2Fapp.component.ts](https://stackblitz.com/edit/angular-4im6ksuc?file=src%2Fapp%2Fapp.component.ts)
